addon/components/sl-split-grid.js

```diff
@@ -0,0 +1,564 @@
+                this.set( 'activeRecord', null );
+            }
+
+            this.set( 'detailsOpen', false );
+        },
+
+        /**
+         * Open the details pane with a specific row object
+         *
+         * @function actions.openDetailsPane
+         * @param    {object} row - The object that the clicked row represents
+         * @returns  {void}
+         */
+        openDetailsPane: function( row ) {
+            var activeRecord = this.get( 'activeRecord' );
+
+            if ( activeRecord ) {
+                Ember.set( activeRecord, 'active', false );
+            }
+
+            Ember.set( row, 'active', true );
+            this.set( 'activeRecord', row );
+
+            this.set( 'detailsOpen', true );
+
+            if ( this.get( 'autoHeight' ) ) {
+                Ember.run.next( this.updateDetailContentHeight.bind( this ) );
+            }
+        },
+
+        /**
+         * Toggle sorting of the selected column, and send the "sortAction"
+         * bound action the column and direction to sort
+         *
+         * @function actions.sortColumn
+         * @param    {object} column - The column definition for the triggered header's column
+         * @returns  {void}
+         */
+        sortColumn: function( column ) {
+            var columnTitle       = Ember.get( column, 'title' ),
+                sortedColumn      = this.getSortedColumn(),
+                sortedColumnTitle = this.get( 'sortedColumnTitle' ),
+                sortedDirection   = this.get( 'sortedDirection' ),
+                direction;
+
+            if ( sortedColumnTitle === columnTitle ) {
+                direction = sortedDirection === 'ascending' ? 'descending' : 'ascending';
+            } else {
+                if ( sortedColumn ) {
+                    Ember.set( sortedColumn, 'sorted' );
+                }
+
+                this.set( 'sortedColumnTitle', columnTitle );
+                direction = 'ascending';
+            }
+
+            this.set( 'sortedDirection', direction );
+            Ember.set( column, 'sorted', direction );
+
+            this.sendAction( 'sortColumn', column, direction );
+        },
+
+        /**
+         * Opens/closes the filter pane
+         *
+         * @function actions.toggleFilterPane
+         * @returns  {void}
+         */
+        toggleFilterPane: function() {
+            this.set( 'filterOpen', !this.get( 'filterOpen' ) );
+
+            if ( this.get( 'autoHeight' ) ) {
+                Ember.run.next( this.updateContentHeight.bind( this ) );
+            }
+        }
+    },
+
+    // -------------------------------------------------------------------------
+    // Events
+
+    // -------------------------------------------------------------------------
+    // Properties
+
+    /**
+     * The text label for the rows' actions buttons
+     *
+     * @property {string} actionsButtonLabel
+     * @default  "Actions"
+     */
+    actionsButtonLabel: 'Actions',
```

Comment:
This needs to be abstracted or definable via component usage to support translation needs